honeybee/cli/create.py

```diff
@@ -0,0 +1,207 @@
+"""honeybee model creation commands."""
+
+try:
+    import click
+except ImportError:
+    raise ImportError(
+        'click is not installed. Try `pip install . [cli]` command.'
+    )
+
+from ladybug_geometry.geometry2d.pointvector import Point2D, Vector2D
+from ladybug_geometry.geometry2d.polygon import Polygon2D
+from ladybug_geometry.geometry3d.pointvector import Point3D, Vector3D
+from ladybug_geometry.geometry3d.face import Face3D
+from ladybug_geometry.geometry3d.polyface import Polyface3D
+from ladybug_geometry_polyskel.polysplit import perimeter_core_subpolygons
+
+from honeybee.model import Model
+from honeybee.room import Room
+from honeybee.facetype import face_types as fts
+from honeybee.boundarycondition import boundary_conditions as bcs
+try:
+    ad_bc = bcs.adiabatic
+except AttributeError:  # honeybee_energy is not loaded and adiabatic does not exist
+    ad_bc = None
+
+import sys
+import os
+import logging
+import json
+import math
+import uuid
+
+_logger = logging.getLogger(__name__)
+
+
+@click.group(help='Commands for creating Honeybee models.')
+def create():
+    pass
+
+
+@create.command('shoe-box')
+@click.argument('width', type=float)
+@click.argument('depth', type=float)
+@click.argument('height', type=float)
+@click.option('--orientation-angle', help='A number between 0 and 360 for the clockwise'
+              'orientation of the box in degrees. (0 = North, 90 = East, 180 = South, '
+              '270 = West).', type=float, default=0, show_default=True)
+@click.option('--window-ratio', help='A number between 0 and 1 (but not equal to 1) '
```

Comment:
Can we also add sill height to give more control in case one wants to do daylight simulation?

Reply:
Yes, there's a separate command that gives control over all that.